resources/js/composables/useS3.ts

```diff
@@ -0,0 +1,22 @@
+export default function () {
+  const uploadFileByPresignedUrl = async ({
```

Comment:
I know this is my code from https://github.com/TitusKirch/saas-template/, so it's a bit weird to give feedback, but here in Portals we don't use objects as wrappers of the Parameter function in the composables. So it should look like this:

```ts
const uploadFileByPresignedUrl = async(
  file: File;
  presignedUrl: string
) {
//...
}
```